src/views/home/Followers.vue

```diff
@@ -1,20 +1,40 @@
 <template>
   <div>
-    <h2>{{ title }}</h2>
+    <v-list>
+      <v-list-item-group color="primary">
+        <FollowingActor
+          v-for="(following, index) in getFollowing"
+          :key="index"
+          :actor="following"
+        />
+      </v-list-item-group>
+    </v-list>
   </div>
 </template>
 
 <script lang="ts">
 import { Component, Vue } from "vue-property-decorator";
+import { namespace } from "vuex-class";
+import { ActivityObject, Link } from "activitypub-objects";
+import FollowingActor from "@/components/following/FollowingActor.vue";
 
-@Component
+const followingStore = namespace("Following");
+
+@Component({
+  components: {
+    FollowingActor
+  }
+})
 export default class MitraFollowers extends Vue {
-  private title = "Followers";
+  private title = "Following";
+
+  @followingStore.Getter
+  public getFollowing!: Array<ActivityObject | Link>;
 }
 </script>
 
 <style lang="scss" scoped>
-div {
+> div {
```

Comment:
This style block needs the following content so that it can only be scrolled within the following content. It is best to define a style class to add in the first DIV

![image](https://user-images.githubusercontent.com/2006124/81064989-fbfe8e80-8eda-11ea-9a9a-ec9691e55b4a.png)

Reply:
Not sure if I understand. Can you make an example what changes are needed from your perspective?

Reply:
```
<template>
  <div class="follower-container">
    <v-list
     ...
    </v-list>
  </div>
</template>

<style lang="scss" scoped>
.follower-container {
  height: 100%;
  overflow: scroll;
  text-align: center;
}

...

</style>
```